banzai/data.py

```diff
@@ -123,12 +123,14 @@ def __imul__(self, value):
         return self
 
     def __itruediv__(self, value):
-        if isinstance(value, CCDData):
-            self.data /= value.data
-            self.uncertainty /= value.uncertainty
+        if isinstance(value, type(self)):
+            self.uncertainty = np.abs(self.data / value.data) * np.sqrt(
+                (self.uncertainty / self.data) ** 2 + (value.uncertainty / value.data) ** 2)
```

Comment:
The spacing on this is weird. maybe do 
self.uncertainty = np.abs(self.data / value.data) * \
    np.sqrt((self.uncertainty / self.data) ** 2 + (value.uncertainty / value.data) ** 2)